src/components/navbar/Navbar.jsx

```diff
@@ -25,30 +25,38 @@ const Navbar = () => {
       {/* TODO change the redirect URL of the icons */}
```

Comment:
probably this todo can be erased after this updates???

Reply:
already deleted